master/lustregittagpoller.py

```diff
@@ -0,0 +1,118 @@
+# -*- python -*-
+# ex: set syntax=python:
+
+import itertools
+import os
+
+from buildbot import config
+from twisted.internet import defer
+from twisted.internet import utils
+from twisted.python import log
+from buildbot.util import epoch2datetime
+from buildbot.changes.gitpoller import GitPoller 
+
+class LustreTagPoller(GitPoller):
+
+    """This source polls a git repo for new tags and generates a change.
+
+    This source will poll a remote git repository for new tags and finds changes
+    to existing tags. Once a new or changed tag is found, a change is submitted to
+    the build master.""" 
```

Comment:
This needs more detail.  How do you determine which tags are "new" or "changed"?  How does this persistently track the list of seen tags?